packages/@angular/cli/commands/build.ts

```diff
@@ -25,6 +25,12 @@ export const baseBuildCommandOptions: any = [
   { name: 'locale', type: String },
   { name: 'extract-css', type: Boolean, aliases: ['ec'] },
   {
+    name: 'compress',
+    type: Boolean,
+    default: true,
```

Comment:
The option will be ignored when -t=development, see https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/models/webpack-config.ts#L59